front/tests/global-setup.ts

```diff
@@ -31,39 +32,38 @@ async function createDataForTests() {
   const rollingStock1 = await postApiRequest('/rolling_stock/', {
     ...exampleRollingStock,
     name: `${exampleRollingStock.name} ${uuidv4()}`,
-  });
+  } as RollingStockUpsertPayload);
```

Comment:
I think you will have to rebase because Tristram migrated the rollingStock types

Reply:
done